Bottom: 41f75e82ccf7f9024c8ee1046d938b1b489c91de
Top:    fd91be76d9514657df03f58ea758fd27ca10b3b7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 20:28:22 +0000

Refresh of update-graphics-api-a-little

---

diff --git a/src/hid_draw.h b/src/hid_draw.h
index df6eb06..407bba1 100644
--- a/src/hid_draw.h
+++ b/src/hid_draw.h
@@ -247,13 +247,13 @@ hid_draw_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 inline void
 hid_draw_pcb_pad (hidGC gc, PadType *pad, bool clip, bool mask)
 {
-  gc->hid_draw->draw_pcb_pad (gc, pad, clip, mask);
+  gc->hid_draw->klass->draw_pcb_pad (gc, pad, clip, mask);
 }
 
 inline void
 hid_draw_pcb_pv (hidGC fg_gc, hidGC bg_gc, PinType *pv, bool draw_hole, bool mask)
 {
-  fg_gc->hid_draw->draw_pcb_pv (fg_gc, bg_gc, pv, draw_hole, mask);
+  fg_gc->hid_draw->klass->draw_pcb_pv (fg_gc, bg_gc, pv, draw_hole, mask);
 }
